api/handler.go

```diff
@@ -558,6 +562,29 @@ func (h *apiHandler) getTotalViews(rw http.ResponseWriter, r *http.Request) {
 	respondJson(rw, http.StatusOK, totalViews)
 }
 
+func (h *apiHandler) queryRealtimeServerViewership() http.HandlerFunc {
+	return func(rw http.ResponseWriter, r *http.Request) {
+		userId := r.URL.Query().Get("userId")
```

Comment:
Could we allow querying by `playbackId` here? Would make it super easy to migrate the existing per-stream `rate_limit` to this if so

Reply:
Yes, we can, but let's leave it as a separate task. I've created a Linear ticket about it: https://linear.app/livepeer/issue/ENG-2120/change-rate-limite-to-viewer-limit